crates/scroll/payload/src/base_fee.rs

```diff
@@ -0,0 +1,89 @@
+use alloy_primitives::U256;
+use revm::{database::State, Database};
+use scroll_alloy_evm::curie::L1_GAS_PRICE_ORACLE_ADDRESS;
+
+/// L1 gas price oracle base fee slot.
+pub const L1_BASE_FEE_SLOT: U256 = U256::from_limbs([1, 0, 0, 0]);
```

Comment:
For better readability, why not just use `U256::from(1)`?

Reply:
You need to use a function marked `const`, that can be evaluated at compile time. The only one available for `U256` is `from_limbs` unfortunately.